template/apps/thanos.json

```diff
@@ -2,20 +2,20 @@
 	"categories": [
 		"Other",
 		"Tools",
-		"Minecraft"
+		"Games"
 	],
   "env": [
   	{
   		"name": "TZ",
   		"label": "TZ",
-  		"default": "America/New_York",
-  		"description": "Specify a timezone to use for example Europe/London"
+  		"default": "Europe/London",
+  		"description": "Specify a timezone to use. For example, Europe/London"
   	}
   ],
-	"description": "[Thanos](https://github.com/aternosorg/thanos) is a tool that allows you to shrink your minecraft worlds by deleting unused chuncks.",
+	"description": "Thanos is a tool that allows you to shrink your minecraft worlds by deleting unused chuncks.",
 	"image_amd64": "cristianeduardmihai/thanos:latest",
-	"image_arm64": "cristianeduardmihai/thanos:arm64",
-	"logo": "https://raw.githubusercontent.com/novaspirit/pi-hosted/images/aternos-thanos.png",
```

Comment:
I _think_ the 404 was just caused by the missing branch name in the original image path, because the logo **is** present [here](https://github.com/novaspirit/pi-hosted/blob/master/images/aternos-thanos.png).

Updating the path to this should fix it:
```
https://raw.githubusercontent.com/novaspirit/pi-hosted/HEAD/images/aternos-thanos.png
```

_(using `HEAD` instead of `master` means it will work even if default branch changed)_

Reply:
Changed it in https://github.com/novaspirit/pi-hosted/pull/464/commits/572927d7ec5aa51fef12a657b5a37e160dc5f9a5